easybuild/easyconfigs/l/launcher/launcher-3.7-GCC-9.3.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,68 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Felipe Franco <felipe.franco@hpcnow.com>, Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'launcher'
+version = '3.7'
+local_python_ver = '3.8.2'
+versionsuffix = '-Python-%s' % local_python_ver
```

Comment:
No need for using a local variable `local_python_ver`: you can just hardcode the `Python` version in `dependencies, and use the `%(pyver)s` template here:

```python
versionsuffix = '-Python-%(pyver)s'
```